core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.spec.ts

```diff
@@ -290,7 +290,7 @@ describe('Translation Suggestion Review Modal Component', function() {
       expect(contributionAndReviewService.reviewExplorationSuggestion)
         .toHaveBeenCalledWith(
           '1', 'suggestion_1', 'accept', 'Review message example: ' +
-          'This suggestion was submitted with reviewer edits.',
+          '(Note: This suggestion was submitted with reviewer edits.)',
```

Comment:
Can we also add a test for this issue's bug, i.e. when the reviewer does not provide their own review message after having edited the suggestion?

Reply:
@sagangwee I'll get started on that. Thank you for reminding me.

Reply:
@sagangwee Done